app/locales/ru.json

```diff
@@ -1,193 +1,193 @@
 {
   "main": {
     "toolTipHeader": "Stretchly - напоминания о перерывах",
-    "microbreakIn": "Mini Break in $t(utils.seconds, {'count': {{seconds}} })",
-    "breakIn": "Long Break in $t(utils.seconds, {'count': {{seconds}} })",
-    "resumingBreaks": "Работа возобновлена",
+    "microbreakIn": "Мини-перерыв через $t(utils.seconds, {'count': {{seconds}} })",
+    "breakIn": "Длинный перерыв через $t(utils.seconds, {'count': {{seconds}} })",
+    "resumingBreaks": "Продолжаем перерывы",
     "downloadLatestVersion": "Загрузить последнюю версию",
-    "toBreak": "Long Break",
-    "toMicrobreak": "Mini Break",
+    "toBreak": "длинному перерыву",
```

Comment:
Lowercased and assuming continuation of `skipToTheNext`

Reply:
Yep, it's item in Tray menu, Skip to next, Long/Mini break